### PR TITLE
Traverse media-query rules.

### PR DESCRIPTION
**Pull request by iamdustan**
Example:

```
.App {
  background-image: url(images/background.png);
}

@media
only screen and (-webkit-min-device-pixel-ratio: 2),
only screen and (        min-device-pixel-ratio: 2),
only screen and (                min-resolution: 192dpi),
only screen and (                min-resolution: 2dppx) {
  .App {
    background-image: url(images/background@2x.png);
  }
}
```

### CODE REVIEW

File: lib/plugins/urls.js [hunk 1]
```diff
@@ -5,10 +5,14 @@ var addImport = require('./../add-import');
 var stripQuotes = require('./../strip-quotes');
 var mark = require('./../mark');
 
-module.exports = function(ast, reworkInstance) {
+function transform(ast, reworkInstance) {
   ast.rules.forEach(function(rule) {
-    if (!rule.declarations)
+    if (!rule.declarations) {
+      if (rule.type === 'media') {
+        process(rule, reworkInstance);
```
Comment (aaronj1335) [inline lib/plugins/urls.js]: it's been a minute since i touched this code, so i'm still jogging my memory... but is this supposed to be `transform`?

Comment (iamdustan) [inline lib/plugins/urls.js]: You are correct. That’s what I get for writing the code locally and then modifying in githubs editor.

**Comment by iamdustan** [comment]
Fixed the typographic error. This should be all green now.

**Comment by aaronj1335** [comment]
ah good to go. sorry for the delay, i'm slammed at work. if you've got a directory you could share that i could try this out to test it i'd be able to merge it faster (it's been a minute since i was using this in any environments i own).

**Comment by aaronj1335** [comment]
merged to master, thanks so much!

**Comment by iamdustan** [comment]
ah thanks! I meant to write up a test case for you but totally forgot in the midst of the week. Thanks!

**Pull request closed without merging**